tests/acceptance/steps/user-identity-assertions.steps.ts

```diff
@@ -0,0 +1,105 @@
+import { Then } from "@cucumber/cucumber";
+import { WorldDefinition } from "./base-verbs.step";
+import assert from "assert";
+
+Then<WorldDefinition>("the status code should be {int}", function (statusCode: number) {
+  assert.ok(this.userIdentityPostResponse);
+  assert.equal(this.userIdentityPostResponse.statusCode, statusCode);
+});
+
+Then<WorldDefinition>("the error should be {string}", function (error: string) {
+  assert.ok(this.userIdentityPostResponse);
+  assert.equal(this.userIdentityPostResponse.body.error, error);
+});
+
+Then<WorldDefinition>("the error description should be {string}", function (errorDescription: string) {
+  assert.ok(this.userIdentityPostResponse);
+  assert.equal(this.userIdentityPostResponse.body.error_description, errorDescription);
+});
+
+Then<WorldDefinition>("the stored identity should be returned", function () {
+  // Build expected credentials array from all credentials
+  const expectedCredentials = this.credentialJwts.map((jwt) => jwt.split(".").at(-1));
+
+  assert.deepEqual(
+    {
+      ...this.userIdentityPostResponse?.body,
+      content: {
+        ...this.userIdentityPostResponse?.body?.content,
+        vot: undefined,
+      },
+      vot: undefined,
+      isValid: undefined,
+      expired: undefined,
+    },
+    {
+      content: {
+        sub: this.userId,
+        iss: "http://api.example.com",
+        credentials: expectedCredentials,
+        vot: undefined,
+      },
+      vot: undefined,
+      isValid: undefined,
+      expired: undefined,
+      kidValid: true,
+      signatureValid: true,
+    }
+  );
+});
+
+Then("the stored identity content.vot should be {string}", function (vot: string) {
+  assert.equal(this.userIdentityPostResponse?.body?.content?.vot, vot);
+});
+
+Then("the stored identity VOT should be {string}", function (vot: string) {
+  assert.equal(this.userIdentityPostResponse?.body?.vot, vot);
+});
+
+Then("the stored identity isValid field is {boolean}", function (isValid: boolean) {
+  assert.equal(this.userIdentityPostResponse?.body?.isValid, isValid);
+});
+
+Then("the stored identity expired field is {boolean}", function (isExpired: boolean) {
+  assert.equal(this.userIdentityPostResponse?.body?.expired, isExpired);
+});
+
+Then<WorldDefinition>("the stored credentials should be returned", function () {
+  // TODO: To be implemented in SPT-1629
+});
```

Comment:
This step has actually been deleted on `main` - I don't think it is actually used anywhere. I think I didn't notice in the original review sorry!